pkgs/top-level/all-packages.nix

```diff
@@ -13381,6 +13381,8 @@ with pkgs;
     libcdio = libcdio082;
   };
 
+  abcmidi = callPackage ../applications/audio/abcmidi { };
```

Comment:
If I understand it correctly then abcmidi is mostly a collection of command line tools. If so then I would suggest to use `../tools/audio/abcmidi` instead.

Reply:
You're absolutely correct. Corrected that.
Meanwhile, I don't really get the ordering in `all-packages.nix`.

Reply:
It is … complicated. Basically, stick your thing in roughly the right place and don't worry about it.